analysis/create_online_learning_curves.py

```diff
@@ -0,0 +1,94 @@
+"""Create plots for online learning."""
+
+import os
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+import pandas as pd
+from predicators.analysis.analyze_results_directory import create_dataframes
+
+pd.options.mode.chained_assignment = None  # default='warn'
+
+############################ Change below here ################################
+
+# X_KEY is the name of the metric that will be plotted on the x axis. See
+# analyze_results_directory.py for all available metrics.
+# X_LABEL is used to label the x axis.
+X_KEY = "CYCLE"  # common: CYCLE, NUM_TRANSITIONS, LEARNING_TIME
+X_LABEL = "Num online learning episodes"
+
+# Y_KEY is the name of the metric that will be plotted on the y axes. See
+# analyze_results_directory.py for all available metrics.
+# Y_LABEL is used to label the y axis.
+Y_KEY = "AVG_NODES_CREATED"  # common: NUM_SOLVED, AVG_NODES_CREATED
```

Comment:
could you refactor these key/label things to be tuples? and instead of writing "common" like this, just make a list where we can comment out lines, and loop over all elements of that list. so for instance...
```
X_KEY_AND_LABEL = [
  ("CYCLE", "Num online learning episodes"),
  ("NUM_TRANSITIONS", "Num transitions"),
  # ("LEARNING_TIME", "Learning time in seconds"),
]
```

which should generate plots for both of the first two rows (but not the third because it's commented out)